### PR TITLE
feat(TS-123): Added config files

### PR DESCRIPTION
**Pull request by julijst**
jira-linkas

The following changes have been implemented:

- Added config file
- Updated readme

Reviewers:

- @julijst

### CODE REVIEW

**Review by augasauruss — APPROVED** [review]
OK

**Review by kiryl-kryvicki — APPROVED** [review]
ACCEPTED

**Review by GG-testing — APPROVED** [review]
Approvinu

File: config.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello
```
Comment (egisb) [inline config.yaml]: test

**Review by egisb — CHANGES_REQUESTED** [review]
truksta naujos eilutes

File: config.yaml (hunk 1 above)
Comment (julijst) [reply config.yaml]: Fixed

**Review by egisb — APPROVED** [review]

**Pull request merged**